deploy/charts/harvester/dependency_charts/whereabouts/values.yaml

```diff
@@ -0,0 +1,50 @@
+# Default values for whereabouts.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: ghcr.io/k8snetworkplumbingwg/whereabouts
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v0.5.4-amd64"
```

Comment:
need to add the image values to the https://github.com/harvester/harvester/blob/master/deploy/charts/harvester/values.yaml so it will be packaged for the air-gap env.

Reply:
added.
thanks a lot